sympy/physics/control/lti.py

```diff
@@ -4263,3 +4263,119 @@ def is_controllable(self):
 
         """
         return self.controllability_matrix().rank() == self.num_states
+
+class PIDController(TransferFunction):
+    """
+    A PID Controller class that uses Parallel connection to combine the P, I, and D components.
```

Comment:
This needs more explanation. Show a blow diagram for the PID and what the "derivative filter time constant" is.